settings.py

```diff
@@ -98,9 +98,9 @@
 if WEB_IMAGE_UPLOAD_PATH and not os.path.exists(WEB_IMAGE_UPLOAD_PATH):
     os.mkdir(WEB_IMAGE_UPLOAD_PATH)
 
-REDIS_HOST='redis'
-REDIS_PORT=6379
-REDIS_DB='0'
+REDIS_HOST=os.environ('REDIS_HOST', 'redis')
+REDIS_PORT=os.environ('REDIS_PORT', '6379')
+REDIS_DB=os.environ('REDIS_DB', '0')
```

Comment:
Let's make this use the existing code to configure settings by setting environment variables. To do that, you can add these `REDIS_HOST`, `REDIS_PORT `and `REDIS_DB` to the existing loop in 

```
for envvar in ['SMTP_PORT', 'SMTP_USERNAME', 'SMTP_PASSWORD', 
...
```

And then to set the defaults after the loop, set the defaults if not already set.

```
if not REDIS_HOST:
    REDIS_HOST='redis'
if not REDIS_PORT:
   ...
```

The settings module is clumsy at the moment at setting defaults, but sticking to the convention of having environment variables formatted like `CANARY_REDIS_HOST`  and `CANARY_REDIS_PORT` using existing code, will help us when we refactor this module to make these types of changes easier.

Reply:
Hi @alanakirby,

Just checking you saw this response. We would like to get this merged in 🤘

Reply:
Hi Jay! Its been so long since Ive worked on this project actually. Would your suggestion above allow for storage other than redis?

Reply:
@alanakirby, I'm unsure this commit should attempt to do that. The reason I say this is because we use `redis` specific libraries through the codebase and so the `redis` python is unlikely to work with other storages. Which storages did you have in mind?